tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature

```diff
@@ -871,3 +871,13 @@ Feature: Share by public link
       | lorem.txt           |
       | simple-empty-folder |
       | data.zip            |
+
+  Scenario: assign password to already created public share
```

Comment:
```suggestion
  Scenario: file owner can assign a password to already created public share
```

Reply:
The purpose of this scenario is to check that password can be assigned to already existing public share, so IMO no need to change the description.